rspace/src/test/scala/coop/rchain/rspace/StorageActionsTests.scala

```diff
@@ -432,6 +432,27 @@ trait StorageActionsTests extends StorageActionsBase {
     store.isEmpty shouldBe true
   }
 
+  "A joined consume with the same channel given twice followed by a produce" should
+    "not raises any errors (CORE-365)" in withTestStore { store =>
```

Comment:
If you `import coop.rchain.rspace.examples.AddressBookExample._` we can directly use the the code in the original report in the test.  

I'm not opposed to having another test like this, but we should definitely have the original too (if possible).

Reply:
Created class StorageExamplesTest, with this specific case.